### PR TITLE
Script for retinaNet and Wandb setup fonctional 

### PR DESCRIPTION
**Pull request by MurielleMardenli200**
The training visualization charts can be viewed in the Google slides [here](https://docs.google.com/presentation/d/1vt_HswO6XyfrnzrVROQbCmXoivmnhdgiDcED2zZ3AYs/edit?usp=sharing) (run_14)

NOTE: The modified files on this branch (dev/retinaNet) were taken from the branch (retinaNet) which is to be deleted

### CODE REVIEW

File: detectron2
Comment (hermancollin) [inline detectron2:1]: hum interesting I didn't know about this submodule feature

**Comment by hermancollin** [comment]
ok this looks good but were you able to start a training? What did it look like?

File: src/retinaNet/wanb_trainer.py
Comment (hermancollin) [inline src/retinaNet/wanb_trainer.py:1]: I like how this is structured :+1:

**Comment by MurielleMardenli200** [comment]
> ok this looks good but were you able to start a training? What did it look like?

I updated the description above with the google slides containing the training visualizations!

**Review by hermancollin — APPROVED** [review]

File: notebooks/data_axondeepseg_sem
Comment (hermancollin) [inline notebooks/data_axondeepseg_sem:1]: btw this data is present twice (in notebooks/ and src/). Could we remove one?

**Pull request merged**